setup.py

```diff
@@ -51,6 +51,10 @@
         "tabulate",
         "dataclasses; python_version<'3.7'",
         "requests>=2.20",
+        "pywin32; platform_system=='Windows'",
+        "psutil; platform_system=='Linux'",
```

Comment:
Probably pre-existing, but how about on Mac?

Reply:
These aren't required on mac, it's using bundled macos command line tools to check for the USB devices and doesn't rely on libraries AFAICT.

Reply:
Hmm, we'll need something for BSD probably

Reply:
I made a sticky note for it.

Reply:
Yeah, we'll probably need to implement BSD specific code IIUC.